src/main/java/uk/gov/ons/census/casesvc/service/SurveyService.java

```diff
@@ -30,6 +33,13 @@ public void processMessage(ResponseManagementEvent surveyEvent, OffsetDateTime m
     UacQidLink surveyLaunchedForQid =
         uacService.findByQid(surveyEvent.getPayload().getResponse().getQuestionnaireId());
 
+    Case caze = surveyLaunchedForQid.getCaze();
+
+    if (caze != null && surveyEvent.getEvent().getChannel().equals("RH")) {
```

Comment:
I know we're only expecting these events from RH, but should we be effectively ignoring them if they come from a different channel?

Reply:
It's not ignoring it, it just won't update the column - because we only want to update the column if it's linked to a case & it's from RH.

This was why I wanted the column called something like rh_survey_launched.